_scripts/clone-legal-docs.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+
+# This script clones the `legal-docs` repo, looks for a `<locale>/[name].md` file matching
+# the file name passed into the `copy_md` function, and copies the matching locale directory
+# and file into `public/legal-docs` if present.
+#
+# Additionally, this script outputs `legal-docs/[name]_locales.json` with an array of locales
+# (directories) that the file was found in to make it easy to know what's available instead
+# of traversing directories when determining which locale to show the user.
+
+# ensure the script errors out when a command fails
+set -e
+
+MODULE="$1"
+
+log() {
+    echo "[clone-legal-docs] $1"
+}
+
+error() {
+    log "$1" >&2
+    exit 1
+}
+
+[ -n "$MODULE" ] || error "You need to specify a module."
+
+if [ -n "$FXA_LEGAL_DOCS_SKIP" ]; then
+    log "Skipping legal-docs update..."
+    exit 0
+fi
+
+
+# BEGIN: workaround for missing GNU realpath in BSD/MacOS-X
+backup_ifs="$IFS"
+
+get_realpath_missing() {
+    if hash -- realpath -m 2>/dev/null; then
+        realpath -m "$1"
+        return
+    fi
+
+    local final_path is_first
+    final_path="$(pwd -P)"
+
+    IFS=/
+    is_first=true
+    for path_part in $1; do
+        if [ -z "${path_part}" ]; then
+            if $is_first; then
+                # absolute path
+                final_path=""
+            fi # otherwise, just ignore (e.g. "//")
+            continue
+        fi
+        is_first=false
+
+        if [ "${path_part}" = ".." ]; then
+            # one up
+            final_path="$(dirname "${final_path}")"
+        elif [ "${path_part}" != "." ]; then
+            final_path="${final_path}/${path_part}"
+        fi # otherwise ignore - current directory
+    done
+    IFS="${backup_ifs}"
+
+    echo "${final_path}"
+}
+# END: workaround for missing GNU realpath in BSD/MacOS-X
+
+DOWNLOAD_PATH="${FXA_LEGAL_DOCS_DOWNLOAD_PATH:-legal-docs}"
+SCRIPT_DIR="$(dirname "$0")"
+MODULE_PATH="$(get_realpath_missing "$SCRIPT_DIR/../packages/$MODULE")"
+
+[ -d "$MODULE_PATH" ] || error "The module/package does not exist at: $MODULE_PATH"
+
+# Clone and pull
+cd "$MODULE_PATH"
+
+if [ -n "${FXA_LEGAL_DOCS_COPY_FROM}" ]; then
+    echo "Using LEGAL_DOCS files from: ${FXA_LEGAL_DOCS_COPY_FROM}"
+    [ ! -d "${DOWNLOAD_PATH}" ] || rm -rf "${DOWNLOAD_PATH}"
+    ln -sf "${FXA_LEGAL_DOCS_COPY_FROM}" "${DOWNLOAD_PATH}"
+else
+    # Git repository to clone
+    FXA_LEGAL_DOCS_REPO="${FXA_LEGAL_DOCS_REPO:-https://github.com/mozilla/legal-docs.git}"
+
+    # branch name
+    FXA_LEGAL_DOCS_BRANCH="${FXA_LEGAL_DOCS_BRANCH:-main}"
```

Comment:
You shouldn't be using `main`, it should use `prod`.

Main might contain changes that are staged weeks before becoming effective.

Reply:
Ah, thank you for catching!